cocotb/binary.py

```diff
@@ -29,27 +29,61 @@
 
 import os
 import random
+import re
 import warnings
 
+
+_RESOLVE_TO_0      = "-lL"
```

Comment:
I know this was just moved, but resolving `-` to `0` is wrong. "Don't care" is a synonym for `X`. Something to note in the future.

Reply:
#1487 with your comment is still open.